reconcile/templating/validator.py

```diff
@@ -30,10 +32,30 @@ class TemplateDiff(BaseModel):
     diff: str
 
 
+class TemplateError(BaseModel):
+    template: str
+    error: str
+    lineno: Optional[int] = None
+
+
+def lint_yaml(template_name: str, yaml_to_lint: str) -> list[TemplateError]:
+    # Possibly move this path to the templating configuration schema in the future
+    resource = gql.get_api().get_resource("/yamllint/yamllint.yml")
```

Comment:
i think this should be an integration option indeed. And if not set, it would disable the linting check. This will ease FedRamp setup, not requiring a file somewhere in resources. WDYT?

Reply:
Indeed, I found this the hardest part of this change :smile: 

My idea was to:

- move `.yamllint` from app-interface root into resources folder
- update the hack script to read the file from there
- then read configuration using GQL

In the design doc, there is a specification about templating configuration. I'd move this setting there in the future. WDYT